src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="marketplace_page">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img class="product-card__img" alt="imac" src="./images/imac.jpeg">
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com">
+      <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"   rel="stylesheet">
+     </img>
+      <div class="product-card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <p class="product-card__code">
+        Код товара: 195434
+      </p>
+      <div class="product-card__rate">
+        <div class="stars stars--4">
+          <div class="star_stars"></div>
+          <div class="star_stars"></div>
+          <div class="star_stars"></div>
+          <div class="star_stars"></div>
+          <div class="star_stars"></div>
+        </div>
+        <p class="product-card__reviews">
+          Отзывов: 5
+        </p>
+      </div>
+      <div class="product-card__price">
+        <p class="product-card__label">
+          Цена
+        </p>
+        <p class="product-card__value">
+          69 999 грн
+        </p>
+      </div>
+      <a href="#" class="product-card__buy-button" data-qa="hover">
+          Купить
+      </a>
```

Comment:
Why don't you add a blank line between siblings (elements on the same level)?

Reply:
Can you pls show me an example how it should look?